spec/craft_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/person'
+require './lib/craft'
+
+describe Craft do
+  before(:each) do
+    @craft = Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})
+  end
+
+  it 'exists' do
+    expect(@craft).to be_a(Craft)
+  end
+
+  it 'has a name' do
+    expect(@craft.name).to eq('knitting')
+  end
+
+  it 'has required supplies' do
```

Comment:
These are good additional expectations but I would also include the one given in the interaction pattern:
```
pry(main)> craft.supplies_required
#=> {:yarn=>20, :scissors=>1, :knitting_needles=>2}
```